Add `metric` to list of `--resource-types` (#3020)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/list.md

See [resource selection syntax](node-selection/syntax) for more information on how to select resources in dbt

**Arguments**:

> - `--resource-type`: This flag limits the "resource types" that dbt will return in the `dbt ls` command. By default, all resources are included in the results of `dbt ls` except for analysis.

Reviewer: ```suggestion
- `--resource-type`: This flag restricts the "resource types" returned by dbt in the `dbt ls` command. By default, all resource types are included in the results of `dbt ls` except for the analysis type.
```

Author: Thanks for this suggestion @mirnawong1 !

Two questions:
1. Do you prefer 'dbt ls' over `dbt ls`? If so, I'm curious to hear more why for my edification!
2. How do you feel about this as the final sentence?
    > By default, all resource types are included in the results of `dbt ls` except for the analysis type.

Reviewer: ack! so for sure that was my bad, i prefer `dbt ls` (per style guide too) and i just forgot to use the code apostrophe 🙈 sorry about causing that confusion!

ok i updated your final sentence - what do you think?

Author: Looks great -- committed your suggestion 🙌
